docs/src/capi/tutorial/getting-started.md

```diff
@@ -1 +1,196 @@
-# Install Canonical K8s with Cluster API
\ No newline at end of file
+# Cluster provisioning with CAPI and Canonical K8s
+
+This guide covers how to deploy a Canonical Kubernetes multi-node cluster
+using Cluster API (CAPI).
+
+## Install `clusterctl`
+
+The `clusterctl` CLI tool manages the lifecycle of a Cluster API management
+cluster. To install it, follow the [upstream instructions]. Typically, this
+involves fetching the executable that matches your hardware architecture and
+placing it in your PATH. For example, at the time this guide was written,
+for `amd64` you would run:
+
+```sh
```

Comment:
you can leave out the code specification for normal terminal commands, it defaults to something sane.